package.json

```diff
@@ -3,7 +3,7 @@
     "version": "0.0.1",
     "scripts": {
         "test": "jest --passWithNoTests",
-        "start": "node src/server.js",
+        "start": "react-scripts start",
```

Comment:
This needs to be changed back to `node src/server.js` before we merge to avoid breaking the Heroku deployment.